examples/sanic_aiopg_sqlalchemy_example.py

```diff
@@ -34,40 +30,37 @@
                  sa.Column("pub_date", sa.DateTime))
 
 
-async def get_engine():
-    return await create_engine(connection)
-
 app = Sanic(name=__name__)
-engine = loop.run_until_complete(get_engine())
 
 
-async def prepare_db():
+async def prepare_db(app, loop):
     """ Let's add some data
 
     """
-    async with engine.acquire() as conn:
-        await conn.execute('DROP TABLE IF EXISTS sanic_polls')
-        await conn.execute("""CREATE TABLE sanic_polls (
-                                    id serial primary key,
-                                    question varchar(50),
-                                    pub_date timestamp
-                                );""")
-        for i in range(0, 100):
-            await conn.execute(
-                polls.insert().values(question=i,
-                                      pub_date=datetime.datetime.now())
-                )
+    async with create_engine(connection) as engine:
+        async with engine.acquire() as conn:
+            await conn.execute('DROP TABLE IF EXISTS sanic_polls')
+            await conn.execute("""CREATE TABLE sanic_polls (
+                                        id serial primary key,
+                                        question varchar(50),
+                                        pub_date timestamp
+                                    );""")
+            for i in range(0, 100):
+                await conn.execute(
+                    polls.insert().values(question=i,
+                                          pub_date=datetime.datetime.now())
+                    )
 
 
 @app.route("/")
 async def handle(request):
-    async with engine.acquire() as conn:
-        result = []
-        async for row in conn.execute(polls.select()):
-            result.append({"question": row.question, "pub_date": row.pub_date})
-        return json({"polls": result})
+    async with create_engine(connection) as engine:
```

Comment:
Where is `create_engine` defined?
Do we really want to recommend to create a new engine in each request?